configs/openmrs/initializer_config/ampathforms/lamp_child_nutrition_form.json

```diff
@@ -2,7 +2,7 @@
   "name": "Programme d’aide à l’alimentation des enfants",
```

Comment:
Until we receive validation from client the equivalent of Child Nutrition should be "Malnutrition infantile" or if we refer to the program "Programme de malnutrition infantile"